src/DataLayouts/DataLayouts.jl

```diff
@@ -411,6 +415,100 @@ end
     IFH{SS, Ni}(dataview)
 end
 
+# ======================
+# Data0D DataLayout
+# ======================
+
+Base.size(data::Data0D) = (1, 1, 1, 1, 1)
+
+struct F{S, A} <: Data0D{S}
```

Comment:
Maybe call it `DataF`? I realize it breaks the convention, but calling it `F` seems like a great way to cause confusion.